src/components/menu/Profile.js

```diff
@@ -26,6 +31,14 @@ export default class Profilemenuitem extends Component {
     });
   };
 
+  loggOut = (event) => {
+    event.preventDefault();
+    const { loggOut } = this.props;
+    loggOut();
+    sessionStorage.token = '';
```

Comment:
@EmyRukundo, I would suggest clearing the sessionStorage like `sessionStorage.clear()`. It seems you are re-assigning the token with an empty string instead of clearing it from the sessionStorage, and that statement should be in your logout action.